packages/core-db/src/app/ethereum/ethereum-block-processor.ts

```diff
@@ -50,6 +50,13 @@ export class EthereumBlockProcessor {
     this.subscriptions.add(handler)
 
     provider.on('block', async (blockNumber) => {
+      if (blockNumber < this.earliestBlock) {
+        log.debug(
```

Comment:
Couldn't this indicate a reorg?

Reply:
Yep, but it's assumed that the earliest block will be finalized. If you say "I don't care about blocks before block X" we assume you are being honest and actually do not care about them.